packages/core/README.md

```diff
@@ -95,3 +95,133 @@ return choices.map(getValuePair).map((choice) => (
+// Vind trenger core for å fungere
+@use "@fremtind/jkl-core/vind";
+```
+
+#### API
+
+Du finner igjen disse tingene du kanskje kjenner fra Tailwind:
+
+-   [Flex Direction](https://tailwindcss.com/docs/flex-direction)
+-   [Flex Grow](https://tailwindcss.com/docs/flex-grow)
+-   [Flex Shrink](https://tailwindcss.com/docs/flex-shrink)
+-   [Gap](https://tailwindcss.com/docs/gap)
+-   [Justify Content](https://tailwindcss.com/docs/justify-content)
+-   [Align Items](https://tailwindcss.com/docs/align-items)
+-   [Padding](https://tailwindcss.com/docs/padding)
+-   [Margin](https://tailwindcss.com/docs/margin)
+-   [Space Between](https://tailwindcss.com/docs/space)
+-   [Font Family](https://tailwindcss.com/docs/font-family) (`font-serif` finnes ikke)
+-   [Font Weight](https://tailwindcss.com/docs/font-weight) (kun `font-normal` og `font-bold`)
+
+##### Typografi
+
+Vi har egne utils for typografi som passer med vår egen typografiskala.
+
+```scss
+.title {
+    @include jkl.text-style("title");
+}
+
+.title-small {
+    @include jkl.text-style("title-small");
+}
+
+.h1 {
+    @include jkl.text-style("heading-1");
+}
+
+.h2 {
+    @include jkl.text-style("heading-2");
+}
+
+.h3 {
+    @include jkl.text-style("heading-3");
+}
+
+.h4 {
+    @include jkl.text-style("heading-4");
+}
+
+.h5 {
+    @include jkl.text-style("heading-5");
+}
+
+.body {
+    @include jkl.text-style("body");
+
+    &.font-bold {
+        @include jkl.no-grow-bold;
+    }
+}
+
+.small {
+    @include jkl.text-style("small");
+
+    &.font-bold {
+        @include jkl.no-grow-bold;
+    }
+}
+```
+
+#### Savner du noe fra Tailwind?
+
+Åpne et issue med feature-ønsker, så kan vi finne ut av det sammen.
+
+#### Jeg vil bruke Tailwind for å bygge løsningene mine. Hvordan bruker jeg Jøkul-verdier?
+
+Per nå har vi ikke noen ferdig konfigurasjon som er klar for Tailwind eller andre utility-baserte verktøy, men vi eksporterer alle design tokens som JavaScript.
+
+Du kan importere tokensene i `tailwind.config.js` og bygge opp konfigurasjonen din på den måten:
+
+```js
+import { tokens } from "@fremtind/jkl-core";
+import plugin from "tailwindcss/plugin";
+
+export default {
+    content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
+    theme: {
+        extend: {},
+        colors: tokens.color,
+        spacing: tokens.spacing,
+        fontSize: null,
+        lineHeight: null,
+    },
+    plugins: [
+        plugin(function ({ addComponents }) {
+            addComponents({
+                ".title": tokens.typography.title.base,
+                ".title-small": tokens.typography.titleSmall.base,
+                ".h1": tokens.typography.h1.base,
+                ".h2": tokens.typography.h2.base,
+                ".h3": tokens.typography.h3.base,
+                ".h4": tokens.typography.h4.base,
+                ".h5": tokens.typography.h5.base,
+                ".body": tokens.typography.body.base,
+                ".small": tokens.typography.small.base,
```

Comment:
@joms, kunne du ha testet om dette egentlig funker?

<details>
<summary>tokens.js</summary>

```js
/**
 * Do not edit directly
 * Generated on Fri, 26 May 2023 12:15:33 GMT
 */

export default {
    color: {
        svart: "#000",
        granitt: "#1b1917",
        skifer: "#313030",
        fjellgra: "#444141",
        stein: "#636060",
        svaberg: "#c8c5c3",
        varde: "#e0dbd4",
        dis: "#ece9e5",
        sand: "#f4f2ef",
        snohvit: "#f9f9f9",
        hvit: "#fff",
        suksess: "#c8ecd2",
        feil: "#ffc9b2",
        info: "#d9ddff",
        advarsel: "#fdeab9",
    },
    spacing: {
        "0": "0",
        "2": "0.125rem",
        "4": "0.25rem",
        "8": "0.5rem",
        "12": "0.75rem",
        "16": "1rem",
        "24": "1.5rem",
        "32": "2rem",
        "40": "2.5rem",
        "64": "4rem",
        "104": "6.5rem",
        "168": "10.5rem",
    },
    typography: {
        weight: {
            normal: "400",
            bold: "700",
        },
        font: {
            size: {
                "16": "1rem",
                "18": "1.125rem",
                "20": "1.25rem",
                "21": "1.3125rem",
                "23": "1.4375rem",
                "25": "1.5625rem",
                "26": "1.625rem",
                "28": "1.75rem",
                "30": "1.875rem",
                "36": "2.25rem",
                "44": "2.75rem",
                "56": "3.5rem",
            },
        },
        line: {
            height: {
                "24": "1.5rem",
                "28": "1.75rem",
                "32": "2rem",
                "36": "2.25rem",
                "40": "2.5rem",
                "44": "2.75rem",
                "52": "3.25rem",
                "64": "4rem",
            },
        },
        title: {
            small: {
                fontSize: "2.25rem",
                lineHeight: "2.75rem",
                fontWeight: "400",
            },
            base: {
                fontSize: "3.5rem",
                lineHeight: "4rem",
                fontWeight: "400",
            },
        },
        titleSmall: {
            small: {
                fontSize: "1.875rem",
                lineHeight: "2.25rem",
                fontWeight: "400",
            },
            base: {
                fontSize: "2.75rem",
                lineHeight: "3.25rem",
                fontWeight: "400",
            },
        },
        h1: {
            small: {
                fontSize: "1.625rem",
                lineHeight: "2rem",
                fontWeight: "400",
            },
            base: {
                fontSize: "2.25rem",
                lineHeight: "2.75rem",
                fontWeight: "400",
            },
        },
        h2: {
            small: {
                fontSize: "1.4375rem",
                lineHeight: "2rem",
                fontWeight: "400",
            },
            base: {
                fontSize: "1.75rem",
                lineHeight: "2.5rem",
                fontWeight: "400",
            },
        },
        h3: {
            small: {
                fontSize: "1.3125rem",
                lineHeight: "1.75rem",
                fontWeight: "700",
            },
            base: {
                fontSize: "1.5625rem",
                lineHeight: "2rem",
                fontWeight: "700",
            },
        },
        h4: {
            small: {
                fontSize: "1.125rem",
                lineHeight: "1.5rem",
                fontWeight: "700",
            },
            base: {
                fontSize: "1.3125rem",
                lineHeight: "1.75rem",
                fontWeight: "700",
            },
        },
        h5: {
            small: {
                fontSize: "1rem",
                lineHeight: "1.5rem",
                fontWeight: "700",
            },
            base: {
                fontSize: "1rem",
                lineHeight: "1.5rem",
                fontWeight: "700",
            },
        },
        body: {
            small: {
                fontSize: "1.125rem",
                lineHeight: "1.75rem",
                fontWeight: "400",
            },
            base: {
                fontSize: "1.25rem",
                lineHeight: "2rem",
                fontWeight: "400",
            },
        },
        small: {
            small: {
                fontSize: "1rem",
                lineHeight: "1.5rem",
                fontWeight: "400",
            },
            base: {
                fontSize: "1rem",
                lineHeight: "1.5rem",
                fontWeight: "400",
            },
        },
    },
};
```

</details>

Reply:
I shall test

Reply:
Konfigurasjonen ser i stor grad ut til å fungere, men som du sier så er ikke typografien responsiv. [Det ser ut som det er mulig å få til](https://stackoverflow.com/a/65002174), men spesielt pent er det ikke. Her tror jeg det kan være lurt å se på om det er mulig å gjøre noe smartere.

Reply:
For ettertiden: Responsiv typografi løses med [CSS In JS](https://tailwindcss.com/docs/plugins#css-in-js-syntax). Dette kan enkelt genereres med noe ala 

```js
const responsiveTokenToCssInJss = (rule) => {
  return {
    ...rule.small,
    [`@media (min-width: ${breakpoints.medium}px)`]: {
      ...rule.base,
    },
  };
};

const convertTokenToCssInJss = (token) => {
  return Object.entries(token).map(([key, value]) => {
    return {
      [key]: responsiveTokenToCssInJss(value),
    };
  });
};

const typographyTokens = convertTokenToCssInJss(tokens.typography);
```

Så kan man bruke disse funksjonene til å legge til nye komponenter i Tailwind configen. Dette burde også funke ut av boksen for Uno

```js
  plugins: [
    plugin(function ({ addComponents }) {
      addComponents(typographyTokens);
    }),
  ],
```

Reply:
Oppdaterte docs med et liknende eksempel